DigitalLearningSolutions.Web.Tests/Helpers/RegisterAdminHelperTests.cs

```diff
@@ -16,7 +16,7 @@ public class RegisterAdminHelperTests
         private const string DefaultCentreEmail = "centre@email.com";
         private IUserDataService userDataService;
         private ICentresDataService centresDataService;
-        private RegisterAdminHelper registerAdminHelper;
+        private IRegisterAdminHelper registerAdminHelper;
```

Comment:
These three (userDataService, centresDataService, registerAdminHelper) should be `= null!` to make Rider's warning go away